desktop/core/src/desktop/templates/scheduler/submit_job_popup.mako

```diff
@@ -168,6 +172,14 @@ else:
     huePubSub.subscribeOnce('hide.datepicker', function () {
       _el.datepicker('hide');
     });
+    $(document).on("click.hideDatepicker", function (event) {
```

Comment:
This attaches an event listener on every click ($(".calendera-link").on("click" ... above). Better to attach it on show and detach on hide.

Reply:
made code changes